app/mails/management/commands/sendmail.py

```diff
@@ -39,61 +31,17 @@ def handle(self, *args, **kwargs):
 
                 return
 
-            charset = message.msg.get_content_charset()
             recipients = mail.recipient_set
             tpl = get_template("mails/messages/mail_attachment.txt")
             text = tpl.render({"recipients": recipients})
+            msg = None
 
-            if message.msg.is_multipart():
-                add_text = MIMEText(text, "plain", "utf-8")
-                if message.msg.get_content_maintype == "multipart/signed":
-                    # If it's a signed message, only take first payload
-                    msg = MIMEMultipart()
-                    orig = message.msg.get_payload(0)
-                    msg.attach(orig)
-                    msg.attach(add_text)
-                else:
-                    msg = MIMEMultipart()
-                    msg.attach(message.msg)
-                    msg.attach(add_text)
-
-            else:
-                msg = MIMEText(
-                    "\n\n".join((message.msg.get_payload(), str(text))),
-                    "plain",
-                    charset,
-                )
+            send_email_with_attachments(message=message, mail=mail, text=text, msg=msg)
```

Comment:
Sure you're calling the right function here?

Also, `msg` is always `None`, why even pass it?

Reply:
wrong function, changed it to the correct one

Reply:
also forgot to make `msg` global, it's used in another function as well